lib/utils/constant.dart

```diff
@@ -0,0 +1 @@
+const pokeImage = "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png";
```

Comment:
can you add TODO comment that this will be removed once dynamic data is used here? 🙂 

also move this string constant to your `strings.dart` and delete this file since this is YAGNI atm 🙈